QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -45,7 +48,8 @@ public void autonomousMain() throws AutonomousModeOverException, InterruptedExce
             Rollers.open.set(false);
 
             waitUntil(crateInPosition);
-
+            wait((long) timeout.get());
```

Comment:
Also, `wait` is not the right method here. You might want `waitForTime`, which can also take a FloatInputPoll directly. (In seconds.)